components/avatar/avatar.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../../style.css">
+    <title>avatar</title>
+</head>
+<body>
+    <div class="avatar avatar-lg">
+        <img class="img-responsive img-round" src="../assets/images/avatar.png" alt="Avatar">
+    </div>
+    <div class="avatar avatar-md">
+        <img class="img-responsive img-round" src="../assets/images/avatar.png" alt="Avatar">
+    </div>
+    <div class="avatar avatar-sm">
+        <img class="img-responsive img-round" src="../assets/images/avatar.png" alt="Avatar">
+    </div>
+    <div class="avatar avatar-xs">
+        <img class="img-responsive img-round" src="../assets/images/avatar.png" alt="Avatar">
+    </div>
+    <div class="avatar avatar-text-lg avatar-text img-round" role="img" alt="Avatar">AB</div>
+    <div class="avatar avatar-text-md avatar-text img-round" role="img" alt="Avatar">AB</div>
+    <div class="avatar avatar-text-sm avatar-text img-round" role="img" alt="Avatar">AB</div>
+    <div class="avatar avatar-text-xs avatar-text img-round" role="img" alt="Avatar">AB</div>
+    <div class="avatar avatar-lg">
+        <img class="img-responsive img-square" src="../assets/images/avatar.png" alt="Avatar">
+    </div>
+    <div class="avatar avatar-md">
+        <img class="img-responsive img-square" src="../assets/images/avatar.png" alt="Avatar">
+    </div>
+    <div class="avatar avatar-sm">
+        <img class="img-responsive img-square" src="../assets/images/avatar.png" alt="Avatar">
+    </div>
+    <div class="avatar avatar-xs">
+        <img class="img-responsive img-square" src="../assets/images/avatar.png" alt="Avatar">
+    </div>
+    <div class="avatar avatar-text-lg avatar-text img-square" role="img" alt="Avatar">AB</div>
+    <div class="avatar avatar-text-md avatar-text img-square" role="img" alt="Avatar">AB</div>
+    <div class="avatar avatar-text-sm avatar-text img-square" role="img" alt="Avatar">AB</div>
+    <div class="avatar avatar-text-xs avatar-text img-square" role="img" alt="Avatar">AB</div>
```

Comment:
same, check for img-square also

Reply:
same, to make it square and to give border